vagrant/guest/bin/vhosts.sh

```diff
@@ -155,6 +190,10 @@ function process_site {
     # if no hostnames are remaining, return to caller
     [[ -z ${site_hosts[@]} ]] && return
 
+    # If no site root can be determined we should return before attempting to build certs and configs
+    local site_pub=$(ls -1dU "$site_path"/{pub,html,htdocs} 2>/dev/null | head -n1)
```

Comment:
Removing this because it is a feature - custom site config files in a site directory should work where there is no pub directory present at one of the default locations (internally at CL we have one of these as a currently active client).